bridge/github/datakit_github.ml

```diff
@@ -95,6 +95,7 @@ module Commit = struct
 
   type t = { repo: Repo.t; id : string }
 
+  let create repo id = {repo; id = String.trim id }
```

Comment:
Adding or not a newline at the end of a file should not break the bridge. I think `String.trim` is sensible here, as it is already done for `PR.create` where the description is truncated to fit GH length constraints.